### DISCUSSION TITLE
Curated Feed

### DISCUSSION THREAD

**Discussion by martingz** [FreshRSS/FreshRSS#3505] [Ideas]
It would be great to have a Curated Feed, where tagged articles could be posted as an RSS feed.

**Comment by Alkarex** [comment]
Hello,
This is already possible: if you define a token in your user's settings, any FreshRSS view can be shared again as an RSS feed.
This includes a view for a given tag, or a search query.

See https://github.com/FreshRSS/FreshRSS/issues/3178
> However, this token approach is on the road-map to be revisited, because the current token is not so much meant for being public, but rather enable the output to be consumed by something else you control.
Indeed, there is currently one single token per user, which basically gives access to all your outputs if you make it public.

**Reply by martingz** [reply]
Oh great! Thanks, I did see that referenced post before I asked, but I thought it meant all feeds. I did not know I could create a view based on a tag - that is perfect (I'll dig that out). 

(And I didn't get yelled at either -:)

**Reply by martingz** [reply]
In case someone else runs across this, to RSS a Label 

- Under Profile, add an authentication token.
- Back on the main screen click the label you want under My labels (you can do this with the fav folder too).
- Look in the URL for a "get=xyz". 
- Add that get=xyz to a copy of the token URL with an &get=xyz in your RSS reader or webapp.

(hopefully I got that right)

**Reply by aledeg** [reply]
There is an easier way to grab the URL. There is the RSS icon in the menu that gives you just that.
There is no need to fiddle with copy and paste :)

**Reply by martingz** [reply]
Thanks!
